modules/Quiz/resources/css/quiz.css

```diff
@@ -372,6 +372,9 @@ button.qContinue, button.qApplied {
     vertical-align: middle;
     color: rgb(78, 166, 240);
 }
+.hint-why-box.smaller-font {
+    font-size:1vw;
+}
```

Comment:
@MaksymKorniiov  
As this ticket was opened on Russian and French - did we validate these specific languages ?

Reply:
@eitanavgil I checked in quiz.js. If length of value longer than 4 character - I add this class.
In all of lang - text == 4 or lower. If it  bigger  - text will  be going outside of block